sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobLeaseAsyncClient.java

```diff
@@ -121,7 +122,8 @@ public String getLeaseId() {
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<String>> acquireLeaseWithResponse(int duration, RequestConditions modifiedRequestConditions) {
         try {
-            return withContext(context -> acquireLeaseWithResponse(duration, modifiedRequestConditions, context));
+            return withContext(context -> acquireLeaseWithResponse(duration, modifiedRequestConditions, context),
+                STORAGE_TRACING_PROPERTIES);
```

Comment:
Thoughts on changing this from using `withContext` and a `String` constant in Storage Common to making this a helper method in Storage Common?

Reply:
Updated with a `withTracingContext` in Storage Common that will update the returned serviceCall context with `STORAGE_TRACING_PROPERTIES`.